### DISCUSSION TITLE
Is there a way to use foliate with Svox Pico TTS ?

### DISCUSSION THREAD

**Discussion by Tuxman2** [johnfactotum/foliate#753] [Q&A]
Hello,

the question is in the title.

I would like to use foliate with Svox Pico TTS.

Thanks.

Note: can we open pdf and html file with foliate ?

**Comment by johnfactotum** [comment]
I don't know how Svox Pico TTS works specifically, but see https://github.com/johnfactotum/foliate/wiki#how-to-use-text-to-speech for an example script for gTTS that can probably be adapted for Svox Pico TTS.

> Note: can we open pdf and html file with foliate ?

PDF and HTML are not supported at the moment. For PDF support, see #139.

**Comment by Tuxman2** [comment]
pico2wave can be use for cli command. Pico2wave is a part of Pico TTS.

Here is a script I found on the net (Link here: https://www.blogging-it.com/text-mit-svox-pico-tts-direkt-von-der-kommandozeile-oder-aus-einer-datei-durch-sprachausgabe-wiedergeben/raspberry-pi.html (look at Text aus Datei wiedergeben part)).

Can you help me to adapt it to use with your foliate software ?

Thank you.

**Comment by johnfactotum** [comment]
Maybe try something like this:
```bash
#!/bin/bash
pico2wave --lang=$FOLIATE_TTS_LANG --wave=/dev/stdout "$(cat /dev/stdin)" | aplay - &
trap 'kill $!; exit 0' INT
wait
```

This is just adapted from the gTTS script. I haven't tested it and it might not work...

**Comment by Tuxman2** [comment]
@johnfactotum: Thanks. I will try it. ;-)
